pkg/snapstore/abs_snapstore.go

```diff
@@ -51,56 +52,64 @@ type ABSSnapStore struct {
 }
 
 type absCredentials struct {
-	BucketName     string `json:"bucketName"`
-	SecretKey      string `json:"storageKey"`
-	StorageAccount string `json:"storageAccount"`
+	BucketName         string  `json:"bucketName"`
+	AccountKey         string  `json:"storageKey"`
+	AccountName        string  `json:"storageAccount"`
```

Comment:
I would say that the mismatch between the struct field and the Json would lead to some confusion in the future

Reply:
Alright, I don't particularly have a strong opinion here - I don't mind reverting the rename; however, the fields are somewhat self explanatory so I don't think it'd be an issue.

Reply:
Reverted.